.github/workflows/backend-staging-deploy.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Get latest git tag
         run: |
           git fetch --prune --unshallow
-          echo "API_VERSION=$(git describe --abbrev=0)" >> $GITHUB_ENV
+          echo "API_VERSION=$(git describe)" >> $GITHUB_ENV
```

Comment:
We use `git describe --tags` to get the version string for the frontend https://github.com/dandi/dandi-archive/blob/master/web/vue.config.js#L10, would it make sense to do that here too?

Reply:
What's the benefit of adding `--tags`? As far as I can tell, it just allows the inclusion of "lightweight" tags. From the `git-tag` man docs,

```
Annotated tags are meant for release while lightweight tags are meant for private or temporary
object labels. For this reason, some git commands for naming objects (like git describe) will
ignore lightweight tags by default.
```

It seems that we always use annotated tags, so I'm inclined to just leave it as `git describe`, but I don't know if it'll make much of a difference.

Reply:
Ah, I didn't know it was for lightweight tags - I think we're fine with just `git describe` then. t looks like `auto` always uses annotated tags https://github.com/intuit/auto/blob/main/plugins/git-tag/src/index.ts#L65-L70